arangod/Aql/AqlValue.cpp

```diff
@@ -1639,7 +1641,7 @@ size_t AqlValue::memoryUsage() const noexcept {
       // no need to count the memory usage for the item blocks in docvec.
       // these have already been counted elsewhere (in ctors of AqlItemBlock
       // and AqlItemBlock::setValue)
-      return sizeofDocvec();
+      return sizeof(std::vector<SharedAqlItemBlockPtr>) + sizeof(SharedAqlItemBlockPtr) * _data.docvec->size();
```

Comment:
Merged this because it looked good at a glance, but it doesn't compile at `_data.docvec` is a pointer and thus doesn't have a value_type.
So in reality it would need something like `return sizeof(std::vector<SharedAqlItemBlockPtr>) + sizeof((typename std::remove_pointer<decltype(_data.docvec)>::type())::value_type) * _data.docvec->size();` which IMHO does not improve readability.